### PR TITLE
Add queue state query for places

### LINKED ISSUE #24: API: Zustand der Warteschlange abrufen (Patientenansicht)
HTTP-GET: /places/**place**/queues?personDetails=short

Answer: 200/OK
JSON-Payload: {id: **string**; name: **string**; entries: {id: **string**; number: **number**} [] } []

Answer 4xx/not ok

### PR DESCRIPTION
**Pull request by Gronner**
Closes #24 

Implementation of the patient view query.

### CODE REVIEW

File: backend/digitales_warten.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python3
+import json
```
Comment (ohaz) [inline backend/digitales_warten.py]: move import down

File: backend/digitales_warten.py [hunk 2]
```diff
@@ -52,6 +54,40 @@ def create_queue(place_id):
...
+    person_detail_level = request.args.get('personDetails', None)
+    if 'short' != person_detail_level:
+        abort(404)
+
+    place = Place.query.filter_by(id=place_id).first()
+    if place is None:
+        abort(404)
+    
+    attached_queues = Queue.query.filter_by(place=place).all()
+    logging.warning(attached_queues)
```
Comment (ohaz) [inline backend/digitales_warten.py]: remove unnecessary log

File: backend/digitales_warten.py [hunk 3]
```diff
@@ -52,6 +54,40 @@ def create_queue(place_id):
...
+    if not len(attached_queues):
+        logging.warning("no_queues_found")
```
Comment (ohaz) [inline backend/digitales_warten.py]: here too

File: backend/digitales_warten.py [hunk 4]
```diff
@@ -52,6 +54,40 @@ def create_queue(place_id):
...
+    for attached_queue in attached_queues:
+        queue_entries = []
+        waiting_entries = Slot.query.filter_by(queue=attached_queue).all()
+        if not len(waiting_entries):
+            for waiting_entry in waiting_entries:
+                entry = { 'id': waiting_entry.id,
+                           'ticketNumber': waiting_entry.ticket_number
+                        } 
+                queue_entries.append(entry)
+        queue = { 'id': attached_queue.id,
```
Comment (ohaz) [inline backend/digitales_warten.py]: perhaps add an empty line before this one to make it clear that this should be in a different indentation level

File: backend/digitales_warten.py [hunk 5]
```diff
@@ -52,6 +54,40 @@ def create_queue(place_id):
...
+                  'name': attached_queue.name,
+                  'entries': queue_entries
+                }
+        queue_states.append(queue)
+    logging.warning(queue_states)
```
Comment (ohaz) [inline backend/digitales_warten.py]: remove log

File: backend/digitales_warten.py (hunk 1 above)
Comment (Gronner) [reply backend/digitales_warten.py]: Fixed

File: backend/digitales_warten.py (hunk 2 above)
Comment (Gronner) [reply backend/digitales_warten.py]: Fixed

File: backend/digitales_warten.py (hunk 3 above)
Comment (Gronner) [reply backend/digitales_warten.py]: Fixed

File: backend/digitales_warten.py (hunk 4 above)
Comment (Gronner) [reply backend/digitales_warten.py]: Fixed

File: backend/digitales_warten.py (hunk 5 above)
Comment (Gronner) [reply backend/digitales_warten.py]: Fixed

**Pull request merged**